op-signer/gen-local-tls.sh

```diff
@@ -7,39 +7,51 @@ TLS_DIR=$SCRIPT_DIR/tls
 
 version=$(openssl version)
 
-if [[ "$version" != "LibreSSL"* ]] && [[ "$version" != "OpenSSL 1.1"* ]]; then
```

Comment:
I've put up a PR to change `gen-local-tls.sh` to use the `alpine/openssl` docker image. This should help make this script more portable, so we don't need to depend on specific versions of installed TLS libs

Here is the PR https://github.com/ethereum-optimism/infra/pull/255

Reply:
that seems like a good change, i'll review, ty! i'm about to merge this PR however, so we'll need to update your PR once that happens to account for my changes. happy to take that task

Reply:
Thanks @edobry. I've fixed merge conflicts #255